lib/libxx/Kconfig

```diff
@@ -61,6 +61,13 @@ if LIBCXX
 config LIBCXX_EXCEPTION
 	bool "Enable Exception Suppport"
 	default y
+	---help---
+		Enable exception support in libc++.
+
+		NOTE: C++ exception support requires toolchain version 10.3.1. If
+		you are	enabling C++ exception then use docker image version 2.0.0
+		by setting CONFIG_DOCKER_VERSION=2.0.0 or using menuconfig
```

Comment:
`by setting CONFIG_DOCKER_VERSION=2.0.0 or using menuconfig` This could give wrong information.
When we modify a value of some config, we must use the menuconfig.
It should not be `or`.
Using the menuconfig, we should set CONFIG_DOCKER_VERSION with 2.0.0 value.